apps/omg_child_chain/lib/omg_child_chain/block_queue/core.ex

```diff
@@ -116,7 +116,7 @@ defmodule OMG.ChildChain.BlockQueue.Core do
           # Ethereum height at which first block was mined
           chain_start_parent_height: pos_integer(),
           # minimal gap between child blocks
-          minimal_enqueue_block_gap: pos_integer(),
+          block_submit_every_nth: pos_integer(),
```

Comment:
Old implementation was inconsistent with pos_integer() type. SMH

Reply:
whoops, I noticed the comment above is stale lol